packages/flutter/test/cupertino/sheet_test.dart

```diff
@@ -62,6 +62,55 @@ void main() {
     );
   });
 
+  testWidgets('showDragHandle adds a drag handle to the top of the sheet', (
```

Comment:
Can you test, either in this case or in a new case, how viewInsets interacts with `SafeArea`? (Maybe just to verify the position of something.)